src/platforms/python/common/configuration/integrations/pymongo.mdx

```diff
@@ -48,6 +45,29 @@ automatically.
 
 </Note>
 
+## Verify
+
+```python
+import pymongo
+
+def main():
+    sentry_sdk.init(...)  # same as above
+    client = pymongo.MongoClient(DATABASE_HOST, DATABASE_PORT)
+
+    db = client["sentry_test_db"]
+    collection = db["sentry_test"]
+
+    with sentry_sdk.start_transaction(name="testing_sentry"):
+        collection.insert_one({"test": "sentry"})
+        collection.find_one({"test": "sentry"})
+
+main()
+```
+
+This will create a transaction called `testing_sentry` in the Performance section of Sentry and will create spans for the database operations.
+
+It takes a couple moments until the data shows up in Sentry.
```

Comment:
```suggestion
This will create a transaction called `testing_sentry` in the Performance section of [sentry.io](https://sentry.io), and create spans for the database operations.

It takes a couple of moments for the data to appear in [sentry.io](https://sentry.io).
```